cfme/containers/overview.py

```diff
@@ -1,15 +1,40 @@
 # -*- coding: utf-8 -*-
 from functools import partial
-from utils.appliance import Navigatable
+
+from navmazing import NavigateToAttribute
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.node import Node
+from cfme.containers.pod import Pod
+from cfme.containers.service import Service
+from cfme.containers.project import Project
+from cfme.containers.route import Route
+from cfme.containers.container import Container
+from cfme.containers.image_registry import ImageRegistry
+from cfme.containers.image import Image
 from cfme.common import Taggable
+from cfme.web_ui import match_location, StatusBox
 from utils.appliance.implementations.ui import CFMENavigateStep, navigator
-from navmazing import NavigateToAttribute
-from cfme.web_ui import match_location
+from utils.appliance import Navigatable
 
 
 match_page = partial(match_location, controller='container_dashboard', title='Container Dashboards')
 
 
+# TODO: Switch the entire file to support widgetastic
+status_boxs = {
+    Route: StatusBox('Routes'),
```

Comment:
**OPTIONAL**: This is OK, but since this may come up elsewhere, perhaps a better idea is to put a "plural" string on each of these objects. That way you can just access Route.plural, and get back "Routes". We did this before in many places.